.pylintrc

```
@@ -0,0 +1,63 @@
+[MESSAGES CONTROL]
+disable=
+# "F" Fatal errors that prevent further processing
+ import-error,
+# "I" Informational noise
+# "E" Error for important programming issues (likely bugs)
+ no-member,
+ no-name-in-module,
+ raising-bad-type,
+ undefined-variable,
+# "W" Warnings for stylistic problems or minor programming issues
+ anomalous-backslash-in-string,
+ arguments-differ,
+ attribute-defined-outside-init,
+ bad-indentation,
+ broad-except,
+ cell-var-from-loop,
+ deprecated-lambda,
+ fixme,
+ logging-not-lazy,
+ lost-exception,
+ no-init,
+ pointless-string-statement,
+ protected-access,
+ redefined-builtin,
+ redefined-outer-name,
+ relative-import,
+ undefined-loop-variable,
+ unsupported-membership-test,
+ unsubscriptable-object,
+ unused-argument,
+ unused-import,
+ unused-variable,
+# "C" Coding convention violations
+ bad-continuation,
+ bad-whitespace,
+ consider-iterating-dictionary,
+ invalid-name,
+ len-as-condition,
+ line-too-long,
+ missing-docstring,
+ old-style-class,
+ superfluous-parens,
+ trailing-newlines,
+ trailing-whitespace,
+ unidiomatic-typecheck,
+ wrong-import-order,
+ wrong-import-position,
+# "R" Refactor recommendations
+ duplicate-code,
+ inconsistent-return-statements,
+ no-else-return,
+ no-self-use,
+ too-few-public-methods,
+ too-many-branches,
+ too-many-locals,
+ too-many-statements,
+# new for python3 version of pylint
+ useless-object-inheritance
+
+[FORMAT]
+# Maximum number of characters on a single line.
+max-line-length=200
```

We agreed time ago to keep `max-line-length` to 119. See the *.flake8* file
```suggestion
max-line-length=119
```

---

Done